README.md

```diff
@@ -55,11 +55,12 @@ The first symbol of the signature is actually the method receiver in Go terms.
 | `Abs`        | `[a] -> [a]`                |        |   ✓    |        | Return a slice containing the absolute values|
 | `All`        | `[a] -> (a -> bool) -> bool`|   ✓    |   ✓    |    ✓   | Returns true if the predicate applies to all elements in the slice|
 | `Any`        | `[a] -> (a -> bool) -> bool`|   ✓    |   ✓    |    ✓   | Returns true if one or more elements satisfy the predicate|
-| `Average`    | `[a] -> a`                  |         |  ✓     |        | Returns the average of all elements|
+| `Average`    | `[a] -> a`                  |        |  ✓     |        | Returns the average of all elements|
 | `Filter`     | `[a] -> (a -> bool) -> [a]` |   ✓    |   ✓    |    ✓   | Filter the slice based on a predicate|
 | `Head`       | `[a] -> a`                  |   ✓    |   ✓    |    ✓   | Return the first element|
 | `init`       | `[a] -> [a]`                |   ✓    |   ✓    |    ✓   | Returns all elements minus the last|
-| `intercalate`| `[a] -> [[a]] -> [a]`       |   ✓    |   ✓    |    ✓   | Intersperses the slice inbetween the provided 2d-slice |
+| `intercalate`| `[a] -> [[a]] -> [a]`       |   ✓    |   ✓    |    ✓   | Intersperses the slice in between the provided 2d-slice |
```

Comment:
This should be uppercased. (Intercalate). 
Unrelated but Init has the same issue (due to me :P)

Reply:
I'm going to fix it together in some docs revisions I'm doing, going to merge this as it is now. :+1: